Source/Widgets/Viewer/Viewer.js

```diff
@@ -176,6 +180,55 @@ define([
         });
     }
 
+    function enableVRUI(viewer, enabled) {
+        var geocoder = viewer._geocoder;
+        var homeButton = viewer._homeButton;
+        var sceneModePicker = viewer._sceneModePicker;
+        var baseLayerPicker = viewer._baseLayerPicker;
+        var animation = viewer._animation;
+        var timeline = viewer._timeline;
+        var fullscreenButton = viewer._fullscreenButton;
+        var infoBox = viewer._infoBox;
+        var selectionIndicator = viewer._selectionIndicator;
+
+        var display = enabled ? 'none' : 'block';
```

Comment:
@emackey I'm not sure what you mean.  The `Viewer` instance does own the container objects that are being used here.  They are created by the viewer and then passed in to the widget (not to own, but to be populated).  In my opinion there is nothing wrong with toggling the display values here (save for the inline-block problem you pointed out).  However. I think the `inline-block` thing is bug in the Geocoder too. The `cesium-viewer-geocoderContainer` should be block and the `Geocoder` widget itself should have an inline-block div inside the container to ensure that it's elements are aligned properly.  Having a requirement that the container passed into the Geocoder widget be `inline-block` seems like a bug in itself to me.

I agree that having a Viewer property to show/hide the UI elements might make sense, but I think it's outside the scope of this initial pull request.

I'm also not sure we want to hide the selection indicator in VR, wouldn't it still have it's uses?